seclang/parser.go

```diff
@@ -118,8 +119,12 @@ func (p *Parser) evaluate(data string) error {
 	if len(opts) >= 3 && opts[0] == '"' && opts[len(opts)-1] == '"' {
 		opts = strings.Trim(opts, `"`)
 	}
-
-	d, ok := directivesMap[strings.ToLower(directive)]
+	directive = strings.ToLower(directive)
+	if directive == "include" {
+		// this is a special hardcoded case
+		return p.FromFile(opts)
+	}
```

Comment:
I'm not sure this will be enough.

Consider a stupid config like this:

```
$ cat config.conf
include config.conf
```

(yeah, it's a nice recursion).

To avoid this loop, may be you should collect the included files into a list, and check if it exists in that list before include that.

Reply:
Now it validates the file is not requesting itself and there is a limit of 500 includes of recursion